editor/src/clj/editor/code/shader.clj

```diff
@@ -168,8 +165,17 @@
                   :lines lines
                   :resource-ext (resource/type-ext resource)}})])
 
+;; Used for rendering in the editor
 (g/defnk produce-full-source [resource lines]
-  (make-full-source (resource/type-ext resource) lines))
+  (let [source (string/join "\n" lines)
+        resource-ext (resource/type-ext resource)
+        resource-path (resource/path resource)
+        shader-stage (shader-stage-from-ext resource-ext)
+        platform "" ;; ok, since it won't register as a gles platform
```

Comment:
We actually need this to compile properly for the target platform (e.g. the user might switch from an Android target to a macOS target)